### ISSUE TITLE
aruba_module_installer.py fails to run on Ubuntu Server 19.04

### ISSUE THREAD

**Issue by ArubaAndy** [aruba/aruba-ansible-modules#21]
```
aruba@ansible1:~/aruba-ansible-modules/aruba_module_installer$ ls
aruba_module_installer.py  library
aruba@ansible1:~/aruba-ansible-modules/aruba_module_installer$ sudo python aruba                                                                                                             _module_installer.py
There was an issue finding your ansible version.
Please run 'ansible --version' from bash, resolve any errors, and verify version                                                                                                              is release version 2.5 or later.
aruba@ansible1:~/aruba-ansible-modules/aruba_module_installer$ sudo ansible --ve                                                                                                             rsion
ansible 2.8.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/usr/share/my_modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.16 (default, Apr  6 2019, 01:42:57) [GCC 8.3.0]
```

```
aruba@ansible1:~/aruba-ansible-modules/aruba_module_installer/library/plugins$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 19.04
Release:        19.04
Codename:       disco
aruba@ansible1:~/aruba-ansible-modules/aruba_module_installer/library/plugins$

```

**Comment by ArubaAndy** [comment]
Same failure occurs on Ubuntu 18.04.2.

```
aruba@ansible2:/etc/ansible/aruba-ansible-modules/aruba_module_installer$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.2 LTS
Release:        18.04
Codename:       bionic
aruba@ansible2:/etc/ansible/aruba-ansible-modules/aruba_module_installer$ ansible --version
ansible 2.8.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/aruba/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15rc1 (default, Nov 12 2018, 14:31:15) [GCC 7.3.0]
aruba@ansible2:/etc/ansible/aruba-ansible-modules/aruba_module_installer$

```

**Comment by Philippe-Benard** [comment]
I have the same issues. I have the impression that “aruba_module_installer.py” is not compatible with Ansible2.8. 
You can use ansible 2.5 for install module (sudo apt-get install ansible=2.5.1+dfsg-1).

**Comment by rajani-abraham** [comment]
The script try to run 'ansible --version' to find the version. Does that work without 'sudo' for you in ubuntu?

**Comment by ArubaAndy** [comment]
> The script try to run 'ansible --version' to find the version. Does that work without 'sudo' for you in ubuntu?

Yes.

aruba@ansible2:~$ ansible --version
ansible 2.8.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/aruba/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15rc1 (default, Nov 12 2018, 14:31:15) [GCC 7.3.0]
aruba@ansible2:~$

**Comment by rajani-abraham** [comment]
OK, identified the issue. The script checks if Ansible version is 2.7.9 or less (line 142) as that was last tested version.

changing below line from 
if '2.5' <= re_version <= '2.7.9':
to 
if '2.5'  <= re_version :
should work. Let me mark as bug and we will plan to fix.

**Comment by rajani-abraham** [comment]
Update: Even if module installation for ansible 2.8 goes through, running playbooks using switch modules might run into issues. We are actively looking into it. Sorry for the inconvenience.

**Comment by rajani-abraham** [comment]
Update - Issue is with one of the Ansible 2.8 plugins. We are working with Ansible folks - https://github.com/ansible/ansible/issues/57447 and following up with them. Meanwhile try to avoid Ansible 2.8 version for now. 
We are still investigating to unblock with other options.

**Comment by tchiapuziowong** [comment]
Fixed with latest commit f4a14069a350641d56389d4953f2fe13c14d9d87 . In regards to AOS-CX compatibility this is still under investigation and is being tracked by issue #25
